api/metrics/permissions.py

```diff
@@ -14,6 +16,14 @@ class IsInstitutionalMetricsUser(permissions.BasePermission):
 
     def has_object_permission(self, request, view, obj):
         auth = get_user_auth(request)
-        if auth.user.has_perm('view_institutional_metrics', obj.institution):
+        if type(obj) != Institution:
```

Comment:
`assert_resource_type(obj, self.acceptable_models)` is more conventional then this. And you should pretty much never use  `type(obj) != Class`,  use `isinstance` instead